### PR TITLE
Add siblings account

### PR DESCRIPTION
**Pull request by indulekha-deriv**

### CODE REVIEW

File: cypress/e2e/wip/Tradershub/addSiblingsAccounts.cy.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    let country = Cypress.env("countries").CO
+    let nationalIDNum = Cypress.env("nationalIDNum").CO
+    let taxIDNum = Cypress.env("taxIDNum").CO
+    let currency = Cypress.env("accountCurrency").USD
+    beforeEach(() => {
+      localStorage.setItem("config.server_url", Cypress.env("stdConfigServer"))
+      localStorage.setItem("config.app_id", Cypress.env("stdConfigAppId"))
+      cy.c_visitResponsive("/endpoint", "desktop")
+      cy.findByRole("button", { name: "Sign up" }).should("not.be.disabled")
+      cy.c_enterValidEmail(sign_up_mail)
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/Tradershub/addSiblingsAccounts.cy.js:31]: we have a new command for this now  as `cy.c_setEndpoint(signUpMail)`
```suggestion
      cy.c_setEndpoint(signUpMail)
```

File: cypress/e2e/wip/Tradershub/addSiblingsAccounts.cy.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+function getCurrencyList() {
+  return cy.get('div.currency-list__items').find('label').then(($labels) => { 
+  const labelCount = $labels.length
+  return labelCount
+  })
+}
+describe("QATEST-5797, QATEST-5820 - Add siblings accounts", () => {
+    const epoch = generateEpoch()
+    const sign_up_mail = `sanity${epoch}@deriv.com`
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/Tradershub/addSiblingsAccounts.cy.js:21]: this should be camelCase as
```suggestion
    const signUpMail = `sanity${epoch}@deriv.com`
```

**Review by bilalmajid-deriv — CHANGES_REQUESTED** [review]

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]

File: cypress/e2e/wip/Tradershub/addSiblingsAccounts.cy.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  })
+  cy.c_addressDetails()
+  cy.c_completeFatcaDeclarationAgreement()
+  cy.c_addAccount()
+  cy.c_checkTradersHubHomePage()
+  cy.c_closeNotificationHeader()
+  cy.findByTestId('dt_currency-switcher__arrow').click()
+  cy.findByRole('button', { name: 'Add or manage account' }).click()
+  getCurrencyList().then((labelCount) => {
+  cy.get('.dc-modal-header__close').click()
```
[the remark this answers is not in the document]
Comment (indulekha-deriv) [reply cypress/e2e/wip/Tradershub/addSiblingsAccounts.cy.js:54]: There is no testing library found for real account signup header close modal

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**